datanode/metrics/prometheus.go

```diff
@@ -84,6 +84,9 @@ var (
 	// Per table segment creation time.
 	networkHistoryCopiedRowsCounter *prometheus.CounterVec
 	networkHistoryCopyTimeCounter   *prometheus.CounterVec
+
+	batcherAddedEntities   *prometheus.CounterVec
```

Comment:
Just wondering whether a gauge here might be more informative (WRT to monitoring/alerts)

Reply:
Its possible to set an alert on rate (count/time), though the key reason for this metric is to be able to monitor overtime the ratio of events received to flushed, so its got potentially a dual purpose.

Reply:
Yes, but for ratios, wouldn't a summary make the most sense? Rather than a direct division of sums over time, using quantiles over time windows. We'd get a baseline using, say, the 95th percentile over a 15m or even 1h time window, and compare that to the 99th percentile over 5 minutes. WRT alerts, I've found this approach much less cumbersome. Sometimes a lot of events fire, or a lot of updates happen due to a single event (e.g. a market terminates, positions close, expiring orders get cancelled, those are just 2 events, but with lots of entities being flushed). That's a one-off however, and shouldn't raise an alert. For things like that, summaries and histograms are much more useful metric types

Reply:
oh, right sorry I thought you originally suggested to use a gauge? I was replying to that, did you update the original comment or am I going mad?

Reply:
yeah, I originally commented gauge, wanted to edit with strike-through but couldn't remember the markdown for that

Reply:
the reason for the two metrics, added and flushed, is to give some insight on the 'shape' of the data (alerting wasn't the purpose here).  Receiving 1000 updates for the same order in a block results in a very different load on data-node to receiving one update for 1000 different orders (as an extreme comparison) - right now we have no insight into this and so we could see big differences in load which would be essentially be unexplainable with the current stats datanode publishes.